core/src/test/scala/unit/kafka/admin/TopicCommandWithAdminClientTest.scala

```diff
@@ -649,6 +650,55 @@ class TopicCommandWithAdminClientTest extends KafkaServerTestHarness with Loggin
     }
   }
 
+  @Test
+  def testDescribeUnderMinIsrPartitionsWhenReassignmentIsInProgress(): Unit = {
```

Comment:
A comment explaining this test case would be helpful. With min.isr=1, the only way it could be under mn.isr is if the leader fails, but it seems all we are doing is a reassignment? Maybe this would be better with `--under-replicated-partitions`?

Reply:
Thanks for calling this out, it is indeed should be `--under-replicated-partitions`.